ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml

```diff
@@ -10786,18 +11020,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.15"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-ipi-ovn-hypershift-mce-guest-f7
-  reporter_config:
```

Comment:
This is removed by mistake.

Reply:
It is moved to below, you can search it in the page. @jianlinliu

Reply:
I meant the `reporter_config` setting.

Reply:
Weird, this is periodics.yaml file, I could not change it

Reply:
Generally this issue is caused when resolving conflicts, I think you can manually add them back, or copy periodics.yaml from upstream, then re-run `make jobs`.

Reply:
Manually added them @jianlinliu

Reply:
@JianLi-RH Not only this job missed `repoter_config`, many ci jobs are in the same case, you can search `reporter_config` in your PR to find all the deletion of `repoter_config`.